README.md

```diff
@@ -69,6 +69,38 @@ are generated from a central template. To edit one of these files, make an edit
 to its template in this
 [directory](https://github.com/googleapis/synthtool/tree/master/synthtool/gcp/templates/node_library).
 
+## Deployment
+
+The building and deployment process of Flaky.dev has been automated using Cloud Build triggers.  The build steps are outlined in the following files: 
+* [`packages/api/cloudbuild-api.yaml`](./packages/api/cloudbuild-api.yaml) for building the API container
+* ['packages/frontend/deployment/staging/cloudbuild-frontend.yaml'](./packages/frontend/deployment/staging/cloudbuild-frontend.yaml) for building the frontend staging environment
+* ['packages/frontend/deployment/production/cloudbuild-frontend-prod.yaml'](./packages/frontend/deployment/production/cloudbuild-frontend-prod.yaml) for building the frontend production environment
+
+We have implemented a two-pipeline deployment process:
+
+* One pipeline builds our API container and deploys it to Cloud Run
+* One pipeline builds the frontend and deploys it to Firebase Hosting
+
+The following environment variables need to be set within the Cloud Run service directly from the Google Cloud Platform console:
+
+* HEAD\_COLLECTION = name of head Firestore collection
```

Comment:
I would be tempted to put `HEAD\_COLLECTION` etc., in back ticks, it makes them jump out a  bit  more.